libs/application-generic/src/usecases/execute-bridge-request/execute-bridge-request.usecase.ts

```diff
@@ -4,8 +4,19 @@ import {
   NotFoundException,
   BadRequestException,
   HttpException,
+  GatewayTimeoutException,
 } from '@nestjs/common';
-import got, { OptionsOfTextResponseBody, RequestError } from 'got';
+import got, {
```

Comment:
In the era of fetch, do we need to rely on `got`? Got by default gives us retries which I am not very sure if it's a great idea.

Reply:
We will tackle this offline.

Reply:
Elaborating a little, is the thinking that we should handle retries elsewhere? At the queue layer, for example?

I tend to agree, the value we're getting from retries in practice is minimal, especially considering the total retry duration after incorporating backoff is less than 1 minute. It's unlikely an external bridge endpoint would recover from a total network fault within that timespan.